server/src/instant/jdbc/sql.clj

```diff
@@ -300,7 +300,13 @@
                   (apply-postgres-config postgres-config# create-connection?# c#)
                   (with-open [ps# (next-jdbc/prepare c# query# opts#)
                               _cleanup# (register-in-progress create-connection?# ~rw c# ps#)]
-                    (~query-fn ps# nil opts#))
+                    (let [res# (~query-fn ps# nil opts#)]
+                      (when (false? (:return-keys opts#))
```

Comment:
Curious q: why add attributes only when this is false?

Reply:
When return-keys is false, then next-jdbc will return its metadata as the result. Otherwise it returns the result-set and there's no :update-count to add to the query attrs.

I'll look and see if there are some general query stats we can tell next-jdbc to give us or there might be a way to grab them from the prepared-statement (`ps#` in the code).

Reply:
cool, good to know!

Reply:
Updated to get use `.getUpdateCount` on the prepared statement